### ISSUE TITLE
Download using glob pattern

### ISSUE THREAD

**Issue by danielc2013** [stelligent/crossing-go#11]
Background:
There is another ticket which is supposed to allow for multiple file downloads. It would also be beneficial to allow for regex/wildcards.

Depends On: https://github.com/stelligent/crossing-go/issues/10

ToDo:
Extend the existing download functionality to also accept glob-type patterns (*,?,[a-Z], etc) and download all the files matching that pattern. Multiple patterns should also be supported, and if duplicates are found, only upload once
Example:
`crossing-go get s3://bucket-name/**/*.py ./first-glob-location/ s3://second-bucket-name/**/*.go ./second-glob-location/`
